### PR TITLE
feat(one-app-runner): create network and env file

### PR DESCRIPTION
**Pull request by infoxicator**
1. Accepts a new flag: createDockerNetwork
2. Creates a new command: `one-app-runner-test` that starts one-app in the background, using the host and connected to a network ready to run tests
3. Adds a new command: `create-runner-env` that creates a random network name and checks for available ports

### CODE REVIEW

File: packages/one-app-runner/README.md [hunk 1]
```diff
@@ -247,6 +247,34 @@ Or in `package.json`
...
+
+>This option requires `docker-network-to-join` to be used as the network name.
+
+Creates a new [docker network](https://docs.docker.com/engine/reference/commandline/network_create/) using the network name provided.
+
+
+Sample usage:
+
+```bash
+npx @americanexpress/one-app-runner --root-module-name frank-lloyd-root --one-app-version 5.0.0 --module-map-url https://example.com/cdn/module-map.json  --module ../frank-lloyd-root --create-docker-network --doker-network-to-join my-network
```
Comment (mtomcal) [inline packages/one-app-runner/README.md:262]:
```suggestion
npx @americanexpress/one-app-runner --root-module-name frank-lloyd-root --one-app-version 5.0.0 --module-map-url https://example.com/cdn/module-map.json  --module ../frank-lloyd-root --create-docker-network --docker-network-to-join my-network
```

File: packages/one-app-runner/bin/create-runner-env.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  const networkName = `one-app-environment-${uuid()}`;
+
+  await fs.writeFile('.env',
+    `HTTP_PORT=${httpPort}
+HTTP_ONE_APP_DEV_CDN_PORT=${devCdnPort}
+NETWORK_NAME=${networkName}
+HTTP_ONE_APP_DEV_PROXY_SERVER_PORT=${devProxyServer}
+HTTP_METRICS_PORT=${metricsPort}
+  `);
+};
```
Comment (mtomcal) [inline packages/one-app-runner/bin/create-runner-env.js:35]: Will this file need to be cleaned up at any point? Will the Module generator add `.env` to `.gitignore`?

File: packages/one-app-runner/src/startApp.js [hunk 3]
```diff
@@ -91,6 +93,18 @@ module.exports = async function startApp({
   };
 
   const generateModuleMap = () => (moduleMapUrl ? `--module-map-url=${moduleMapUrl}` : '');
+
+  if (createDockerNetwork && dockerNetworkToJoin) {
```
Comment (JAdshead) [inline packages/one-app-runner/src/startApp.js:97]: should provide some feedback if `createDockerNetwork` is `true` but no `dockerNetworkToJoin`

File: packages/one-app-runner/bin/create-runner-env.js (hunk 2 above)
Comment (infoxicator) [reply packages/one-app-runner/bin/create-runner-env.js:35]: yeah it shouldn't be included in git, generator should add it to .gitignore plus is standard practice to always exclude the .env file

File: packages/one-app-runner/src/startApp.js (hunk 3 above)
Comment (infoxicator) [reply packages/one-app-runner/src/startApp.js:97]: "createDockerNetwork flag is true but dockerNetworkToJoin is undefined, please pass a valid network name value"

**Pull request merged**